.mocharc.js

```diff
@@ -1,4 +1,7 @@
+const os = require('os');
+
 module.exports = {
     colors: true,
     'enable-source-maps': true,
+    jobs: Math.min(2, os.cpus().length - 1),
```

Comment:
could a slow/virtual machine return `os.cpus().length === 1`?
which will in turn cause `jobs: 0`

also, this didn't solve it on @barak007 machine

Reply:
It would be the same result as today (https://github.com/mochajs/mocha/blob/34643e4c0821aeb8d6977c1942bc106c9363789a/lib/nodejs/buffered-worker-pool.js#L50)

Reply:
on my machine that failures where timeouts because overload (the meet was open)...This is the 6th time this PR does not fail.